o1vm/tests/test_riscv_elf.rs

```diff
@@ -384,3 +384,33 @@ fn test_divu_by_zero() {
         witness.step();
     }
 }
+
+#[test]
+#[should_panic]
+fn test_rem_by_zero() {
+    let curr_dir = std::env::current_dir().unwrap();
+    let path = curr_dir.join(std::path::PathBuf::from(
+        "resources/programs/riscv32im/bin/rem_by_zero",
+    ));
+    let state = o1vm::elf_loader::parse_riscv32(&path).unwrap();
+    let mut witness = Env::<Fp>::create(PAGE_SIZE.try_into().unwrap(), state);
+
+    while !witness.halt {
+        witness.step();
+    }
```

Comment:
missing assertions

Reply:
No, the idea is that it should fail. See `#[should_panic]`